aztec/src/main/kotlin/org/wordpress/aztec/Aztec.kt

```diff
@@ -21,6 +21,7 @@ open class Aztec private constructor(val visualEditor: AztecText, val toolbar: A
     private var onVideoTappedListener: AztecText.OnVideoTappedListener? = null
     private var onAudioTappedListener: AztecText.OnAudioTappedListener? = null
     private var onMediaDeletedListener: AztecText.OnMediaDeletedListener? = null
+    private var onVideoPressInfoRequestedListener: AztecText.OnVideoPressInfoRequestedListener? = null
```

Comment:
Since we'd like to keep Aztec generic and VideoPress is WP-specific, we could move this to the shortcode plugin (using extension functions or inheritance).